tests/integration/components/battleship-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | battleship', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<Battleship />`);
+
+    assert.equal(this.element.textContent.trim(), '');
```

Comment:
`assert.dom('svg').exists();` ?

Reply:
It's not critical to the function, just looks pretty. Gonna rip the test out.